### PR TITLE
Genaric radio implants are no longer syndicate branded

### PR DESCRIPTION
**Pull request by beck-thompson**
## About the PR

It didn't really make sense that they were syndicate branded! This also resolves the issue of finding a syndicate implant and not being able to do a search because it could be legal. Now all syndicate implants are illegal!

## Media

image

## Requirements

- [X] I have tested all added content and changes.
- [X] I have added media to this PR or it does not require an ingame showcase.

**Changelog**

:cl: Beck Thompson
- tweak: Generic radio implants are no longer syndicate branded.

### CODE REVIEW

**Comment by beck-thompson** [comment]
The implant name and stuff is a little weird (You wont get the name unless you drop it and then pick it up again) but I think that's an unrelated issue

**Comment by deltanedas** [comment]
yeah thats normal

**Comment by TadJohnson00** [comment]
Direction has requested that you remove it from the uplink and at it to the logistics catalogue on the consideration that it lacks syndicate branding and has dubious legality.

**Comment by beck-thompson** [comment]
> Direction has requested that you remove it from the uplink and at it to the logistics catalogue on the consideration that it lacks syndicate branding and has dubious legality.

Execution has acknowledged Directions request and has made the required changes

**Comment by LadyDanger2743** [comment]
> > Direction has requested that you remove it from the uplink and at it to the logistics catalogue on the consideration that it lacks syndicate branding and has dubious legality.
> 
> Execution has acknowledged Directions request and has made the required changes

Beautiful. The Director approves. Something something glory to NanoTrasen.

File: Resources/Prototypes/DeltaV/Catalog/Fills/Crates/medical.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+﻿- type: entity
+  id: CrateGenericRadioImplants
+  parent: CrateMedical
+  name: radio implant crate
+  description: Communicate without having to have that pesky headset on your ear.
```
Comment (deltanedas) [inline Resources/Prototypes/DeltaV/Catalog/Fills/Crates/medical.yml]: needing a
having to have my beloved

**Review by deltanedas — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/DeltaV/Catalog/Fills/Crates/medical.yml (hunk 1 above)
Comment (beck-thompson) [reply Resources/Prototypes/DeltaV/Catalog/Fills/Crates/medical.yml]: Grammar issues are always my downfall 😆

**Review by deltanedas — APPROVED** [review]

**Pull request merged**